crates/astria-sequencer-relayer/src/data_availability.rs

```diff
@@ -46,7 +46,7 @@ use crate::{
 };
 
 pub const DEFAULT_PFD_GAS_LIMIT: u64 = 1_000_000;
-const DEFAULT_PFD_FEE: u128 = 2_000;
+const DEFAULT_PFD_FEE: u128 = 100_000;
```

Comment:
Please split this out into its own PR because these changes are unrelated (even if small).

Reply:
I needed this all in one place so that I could cut a tag and create updated and working images.

I wouldn't have included this here otherwise.

Reply:
Make one branch A with the docker change, then fork a new branch B from A with the gas change.

Open 2 PRs, and for branch B set GitHub to merge into (target) A.

You can now create images on PR B.

Once PR A is merged GitHub will automatically update PR B to merge to main.

Reply:
Wouldn't the final PR get squash merged and ultimately all look the same in main at the end? I'm not sure it's worth all of this effort.

Reply:
No, you have 2 PRs that will result in separate commits, so nothing gets squashed.


Re effort: if you had branched off of your original PR this would be less work.
 
You'll have more work writing the commit message explaining why these changes are necessary. :-)

Reply:
My original PR has main changes.

Reply:
For posterity: discussed in Slack and okay with solution to do all the context around this work.